_data/Iamshankhadeep /Iamshankhadeep.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shankhadeep Dey # No longer than 18 characters
+institution: Siliguri Institute of Technology # no longer than 58 characters
+profile_pic: iamshankhadeep.jpg # Relative path to the image you uploaded
+quote: "And, when you want something, all the universe conspires in helping you to achieve it." # No longer than 100 characters
```

Comment:
Remove double quotes from begin and end.

Reply:
Thank you. and I removed it .